### PR TITLE
fix getThumbnailsByLongestSideSet

### PR DESCRIPTION
**Pull request by atarkowska**
partial rebase of #5062

------------

# What this PR does

fixing getThumbnailsByLongestSideSet in readonly group

https://trello.com/c/3eEBEPG4/7-retrieving-multiple-thumbnails

# Testing this PR

- test thumbnail version like in https://github.com/openmicroscopy/openmicroscopy/pull/3713#issuecomment-95146309

--thumbnails

### CODE REVIEW

**Comment by atarkowska** [comment]
This change probably caused the failure of https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-integration-Python27/446/testReport/OmeroWeb.test.integration.test_tree/TestTree/test_marshal_images_thumb_version/

**Comment by atarkowska** [comment]
test passing now https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-Python27/lastCompletedBuild/testReport/OmeroWeb.test.integration.test_tree/TestTree/test_marshal_images_thumb_version/

**Comment by jburel** [comment]
Removed exclude label

File: components/tools/OmeroWeb/test/integration/test_tree.py [hunk 1]
```diff
@@ -1782,9 +1782,9 @@ def test_marshal_images_thumb_version(self, userA, image_pixels_userA):
         Test marshalling image, loading thumbnail version
         """
         conn = get_connection(userA)
-        # Thumbnail not yet created: version will be -1
+        # Thumbnail is created, see testlib.ITest.createTestImage
         expected = expected_images(userA, [image_pixels_userA],
-                                   extraValues={'thumbVersion': -1})
+                                   extraValues={'thumbVersion': 0})
```
Comment (will-moore) [inline components/tools/OmeroWeb/test/integration/test_tree.py]: If the behaviour of ```testlib.ITest.createTestImage``` has changed then we need to use another way of creating images without a thumbnail to test that the initial version is -1 (e.g. during pyramid generation).

Comment (atarkowska) [reply components/tools/OmeroWeb/test/integration/test_tree.py]: I agree that we should test if view gives correct version on pyramid creation, but `test_marshal_images_thumb_version` never done it before, so if we need more extensive testing they have to be written in line with https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/test/integration/test_thumbs.py.

In this PR I can introduce `with_thumb` flag to `testlib.ITest.createTestImage` and parametrize existing test.

**Comment by jburel** [comment]
looks good

**Comment by atarkowska** [comment]
thx

**Pull request merged**